AspNetCore.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 17
-VisualStudioVersion = 17.0.31324.12
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.31320.298
```

Comment:
Is this intentional?

Reply:
I have unintentionally not yet installed VS17. But I will do that soon. I assume this header flip-flops all the time anyway?